core/templates/css/oppia.css

```diff
@@ -3420,6 +3419,19 @@ oppia-noninteractive-link {
   z-index: 1;
 }
 
+.oppia-static-full-content {
```

Comment:
Let's please avoid introducing new stuff in oppia.css, it's too bloated already. If this is specific to a particular page or component, then keep it in that page/component.

Reply:
Done.

Reply:
Does not seem to be done. (Please check "Files Changed" and make sure it has the changes you want to send before you post your replies to reviews / assign reviewers.)

Reply:
Done.